lambdas/process-candidate-identity/src/test/java/uk/gov/di/ipv/core/processcandidateidentity/service/TicfCriServiceTest.java

```diff
@@ -0,0 +1,288 @@
+package uk.gov.di.ipv.core.processcandidateidentity.service;
```

Comment:
we'll need a consumer test for this lambda and ticf but can probably be done as a separate pr?

Reply:
ahh good point, thanks!